recipe/meta.yaml

```diff
@@ -32,6 +32,7 @@ test:
     - test -f $PREFIX/lib/pkgconfig/{{ name }}.pc  # [unix]
     - test -f ${PREFIX}/lib/cmake/{{ name }}/{{ name }}-config.cmake  # [unix]
     - if not exist %PREFIX%\\Library\\bin\\lzf%SHLIB_EXT% exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\lib\\lzf.lib exit 1  # [win]
```

Comment:
Hard coded `.lib` as I was not able to find a good one from here: https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#environment-variables-set-during-the-build-process